packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -317,7 +317,7 @@ export default function makeKernelKeeper(storage) {
   }
 
   function getVatIDForName(name) {
-    assert(name === `${name}`, details`${name} is not a string`);
+    assert.equal(name, `${name}`, details`${name} is not a string`);
```

Comment:
another `assert.typeof` which might actaully have gotten changed in a later checking.

Reply:
yup. already done.